chat-commands.js

```diff
@@ -1409,6 +1410,52 @@ exports.commands = {
 		user.leaveRoom(targetRoom, connection);
 	},
 
+	getglobal: 'requestglobal',
+	requestglobal: function (target, room, user) {
+		if (!target) return this.parse('/help requestglobal');
+		const staff = Rooms('staff');
+		if (!staff) return this.errorReply("This server does not have a staff room, therefore there is no room to notify of your request.");
+		target = target.trim();
+		const separator = 'answer request id ';
+		const astext = 'text-decoration: underline; color: #0000EE; border: none; background: none; padding: 0; font-family: Verdana,sans-serif; font-size: 12px';
+		if (target.startsWith(separator)) {
```

Comment:
I'm not sure using a separator like this is the most intuitive way to go about this. Maybe something like this would be better:
```javascript
if (!isNaN(+target) && +target >= 0) {
    if (!this.can('lock')) return this.errorReply("Some appropriate error message");
    ...
}
```